src/constants.ts

```diff
@@ -15,3 +16,70 @@ export const routes = {
     getDomainsById: "v1/domains/get/",
   },
 };
+
+export const mappingSortProps: MappingSortProps = {
+  domainId: {
+    domainId: -1,
+  },
+  isRoot: {
+    isRoot: -1,
+  },
+  isValid: {
+    isValid: -1,
+  },
+  registrar: {
+    registrar: -1,
+  },
+  label: {
+    label: -1,
+  },
+  name: {
```

Comment:
Why not keep the domainReflectionSchema and populate this information dynamically?

Reply:
I updated the design of `data-store-api` to filter out invalid sorting fields, and map and pass them to `data-store` in mongodb sorting way, then data-store can use them without changing(i.e. `~.value`).
So it requires to list out all the valid sorting fields of API params.
Please let me know if you have better solution. @Will-Hyde

Reply:
So youre doing this just to support the .value of some fields?

Reply:
yeah, and filtering invalid sorting fields, and collecting all the relevant fields